docs/source/_json/collection.resp

```diff
@@ -71,5 +71,6 @@ Content-Type: application/json
   "sort_reversed": null, 
   "subjects": [], 
   "text": null, 
-  "title": "My Collection"
+  "title": "My Collection", 
+  "version_id": "current"
```

Comment:
@jaroel is there a specific reason why you choosed "version_id" instead of just "version"?